tools/RAiDER/utilFcns.py

```diff
@@ -223,26 +208,24 @@ def writeArrayToRaster(array, filename, noDataValue=0., fmt='ENVI', proj=None, g
     array_shp = np.shape(array)
```

Comment:
I think this section (here and the next several lines) are where bringing along the attributes noted above would help a lot, since then everything is in there and we don't have to name it separately. We simply have one input dictionary that gets assigned when writing the file (new line 224 below).

Reply:
Done.